docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,9 @@ IBM® block storage CSI driver 1.9.0 introduces the enhancements that are detail
 
 **General availability date:** 18 March 2022
 
+## Additional high availability (HA) feature
+
+Stretched volumes and (FlashCopy) snapshots are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```

Comment:
this might sound like snapshots are a new feature. how about:
```suggestion
Stretched volumes and stretched (FlashCopy) snapshots are now supported on SAN Volume Controller storage systems. Stretched storage topology enables disaster recovery and high availability between nodes in I/O groups at different locations. For more information, see **Product overview** > **Technical overview** > **Systems** > **Stretched systems** within the [SAN Volume Controller documentation](https://www.ibm.com/docs/en/sanvolumecontroller).
```

Reply:
DONE